apps/paymaster-proxy/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@eth-optimism/paymaster-proxy",
+  "version": "0.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "typecheck": "tsc --noEmit",
+    "dev": "tsup src/app.ts --watch --onSuccess 'node dist/app.js'",
+    "build": "tsup src/app.ts",
+    "start": "node dist/app.js",
+    "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
+    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.21",
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
+    "eslint": "^8.53.0",
+    "express": "^4.18.2",
```

Comment:
`express` should be a regular dep and not a dev dep

Reply:
fixed!